src/load/azext_load/data_plane/utils/argtypes.py

```diff
@@ -0,0 +1,202 @@
+from azext_load.data_plane.utils import validators, completers
+from azure.cli.core.commands.parameters import (
+    get_resource_name_completion_list,
+    quotes,
+    resource_group_name_type,
+)
+from knack.arguments import CLIArgumentType
+
+quote_text = "Use {} to clear existing {{}}.".format(quotes)
+
+# Common arguments
+resource_group = resource_group_name_type
+
+load_test_resource = CLIArgumentType(
+    options_list=["--load-test-resource", "--name", "-n"],
+    type=str,
+    required=True,
+    completer=get_resource_name_completion_list("Microsoft.LoadTestService/LoadTests"),
+    help="Name or ARM resource ID of the load test resource.",
+)
+#
+
+test_id = CLIArgumentType(
+    validator=validators.validate_test_id,
+    completer=completers.get_test_id_completion_list(),
+    options_list=["--test-id", "-t"],
+    type=str,
+    help="Test ID of the load test",
+)
+
+test_run_id = CLIArgumentType(
+    validator=validators.validate_test_run_id,
+    completer=completers.get_test_run_id_completion_list(),
+    options_list=["--test-run-id", "-r"],
+    type=str,
+    help="Test run ID of the load test run",
+)
+
+existing_test_run_id = CLIArgumentType(
+    validator=validators.validate_test_run_id,
+    completer=completers.get_test_run_id_completion_list(),
+    options_list=["--existing-test-run-id"],
+    type=str,
+    help="Test run ID of an existing load test run",
+)
+
+test_display_name = CLIArgumentType(
+    options_list=["--display-name"],
+    type=str,
+    help="Display name of the load test.",
+)
+
+test_run_display_name = CLIArgumentType(
+    options_list=["--display-name"],
+    type=str,
+    help="Display name of the load test run.",
+)
+
+engine_instances = CLIArgumentType(
+    options_list=["--engine-instances"],
+    type=int,
+    help="Number of engine instances to use for the load test.",
+)
+
+key_vault_reference_identity = CLIArgumentType(
+    options_list=["--key-vault-reference-identity"],
+    type=str,
+    help="The identity that will be used to access the key vault.",
+)
+
+load_test_config_file = CLIArgumentType(
+    options_list=["--load-test-config-file"],
+    type=str,
+    help="Path to the load test config file.",
+)
+
+subnet_id = CLIArgumentType(
+    options_list=["--subnet-id"],
+    type=str,
+    help="ID of the subnet to use for the load test incase of private network.",
+)
+
+test_description = CLIArgumentType(
+    options_list=["--test-description"],
+    type=str,
+    help="Description of the load test.",
+)
+
+test_run_description = CLIArgumentType(
+    options_list=["--description"],
+    type=str,
+    help="Description of the load test run.",
+)
+
+test_plan = CLIArgumentType(
+    options_list=["--test-plan"],
+    type=str,
+    help="Path to the test plan file.",
```

Comment:
Why no validators for this?

Reply:
Need to add validators for path and to check if file exists

Reply:
Added